src/routes/auth/login.ts

```diff
@@ -108,20 +97,17 @@ async function loginAccount({ body }: Request, res: Response): Promise<unknown>
   // generate JWT
   const jwt_token = createHasuraJwt(account)
   const jwt_expires_in = newJwtExpiry
-
-  // return
-  if (useCookie) {
-    res.send({
-      jwt_token,
-      jwt_expires_in
-    })
-  } else {
-    res.send({
-      jwt_token,
-      jwt_expires_in,
-      refresh_token
-    })
+  const user: UserData = {
+    id: account.user.id,
+    display_name: account.user.display_name,
+    email: account.email,
+    avatar_url: account.user.avatar_url
   }
+  let session: Session = { jwt_token, jwt_expires_in, user }
```

Comment:
it reminds me that we may want to clarify our conventions and only use `camelCase` instead of `snake_case` for variable names as it is a common styling recommendation

Reply:
True, we are changing the SDK to use `camelCase` only. The thing is that JSON fields are usually in `snake_case` even though the spec does not enforce one over the other.